docs/network-policy.md

```diff
@@ -168,29 +174,29 @@ The example policy contains a single rule, which allows matched traffic on a
 single port, from one of two sources: the first specified by a `podSelector`
 and the second specified by a combination of a `podSelector` and a
 `namespaceSelector`.
-**Note**: The order in which the ingress rules are set matter, i.e. rules will be
-enforced in the order in which they are written.
+**Note**: The order in which the ingress rules are set matter, i.e. rules will
+be enforced in the order in which they are written.
 
-**egress**: Each ClusterNetworkPolicy may consist of zero or more ordered set of
-egress rules. Each rule, depending on the `action` field of the rule, allows
+**egress**: Each ClusterNetworkPolicy may consist of zero or more ordered set
+of egress rules. Each rule, depending on the `action` field of the rule, allows
 or drops traffic which matches both the `to` and `ports` sections. The example
 policy contains a single rule, which drops matched traffic on a single port,
 to the 10.0.10.0/24 subnet specified by the `ipBlock` field.
-**Note**: The order in which the egress rules are set matter, i.e. rules will be
-enforced in the order in which they are written.
+**Note**: The order in which the egress rules are set matter, i.e. rules will
+be enforced in the order in which they are written.
 
 ### Rule enforcement based on priorities
 
 With the introduction of tiers, ClusterNetworkPolicies are first enforced
 based on the tier to which they are associated with. i.e. all CNP belonging
-to the "Emergency" tier are enforced first, followed by policies associated with
-the "SecurityOps" tier and so on, until the "Application" tier policies are
-enforced. Within a tier, rules belonging to Cluster NetworkPolicy CRDs are
+to the "Emergency" tier are enforced first, followed by policies associated
+with the "SecurityOps" tier and so on, until the "Application" tier policies
+are enforced. Within a tier, rules belonging to Cluster NetworkPolicy CRDs are
```

Comment:
Do you feel these overlap with what you added? Should we just revise this section (or move some contents about tier and policy priority to your section, but keep rule priority here)?

Reply:
revised it in its own section..